### DISCUSSION TITLE
Okta integration with wiki.js

### DISCUSSION THREAD

**Discussion by prasann** [requarks/wiki#4783] [Help / Questions]
I'm trying to integrate the wiki.js with my organization's Okta.

### In wiki.js when i enabled the okta module for authentication i got the following options.

Callback URL / Redirect URI
https://my.domain.com/login/2127e094-84b8-4eef-832c-f39da5ef4744/callback

Login URL
https://my.domain.com/login

Logout URL
https://my.domain.com

### In Okta i have configured accordingly,

Sign-in redirect URIs :
https://my.domain.com/login

Sign-out redirect URIs :
https://my.domain.com

Initiate login URI :
https://my.domain.com/login/2127e094-84b8-4eef-832c-f39da5ef4744/callback

However, when i tried logging in i see a 400 error in Okta with a message,

```
The 'redirect_uri' parameter must be a Login redirect URI in the client app settings: 
```

can someone help me here? 

The okta <-> wiki.js integration doc (https://docs.requarks.io/auth/okta) doesn't talk about the `Initiate login URI`

**Comment by Draawa** [comment]
Hi, I'll allow myself to up this post as I'm encountering the same issue.
@prasann have you ever find a fix though ?

I also have some additional information:
When disabling the https redirection, and then using only http in the URLs (in the okta-side settings), we are able to manually remove the 's' from the URL on our wiki.js frontpage and then by choosing the okta login, it does work and we are redirected to our session, then we can put back the https in the URL.

**Comment by francisbrand** [comment]
I am also having issues with trying to get Okta to work with Wiki.js. Whenever I click Okta on our wiki it opens a page (with the call back URI) with `forbidden` as the only line of text. Tried swapping URI around and also the above using with http rather than https.

I am on a trial account for Okta so had wondered if that was why (can’t find anything saying trial accounts can’t use custom apps). My Wiki is on Synologies Docker with it’s mapped ports exposed.
